packages/web-api/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@slack/web-api",
-  "version": "7.10.0",
+  "version": "7.10.0-aiAppsBeta.5",
```

suggestion: We'll need to revert this version
```suggestion
  "version": "7.10.0",
```

---

🔮 Foretold in https://github.com/slackapi/node-slack-sdk/pull/2399/commits/b8a3ab02052671d728d88515e43a925bf8bf5c7a!